### ISSUE TITLE
Check for duplicates for playing servers array

### ISSUE THREAD

**Issue by R2D2VaderBeef** [The-Mist-Development/The-Mist-Bot#25]
Somewhere in the music.js, I've made an array for which servers are playing. This is how the restart system also contained within that file knows when it's safe to restart the bot and not cut anyone's music. 

In certain situations, like playing a song right when the bot leaves the channel, an error occurs after queueing and the song never plays, the bot never joins the channel. When the song is queued, it added the server id to the list of playing servers.

Now, the server is not playing, but still registered as such in the array. This makes restarting impossible.

When the server is really played, we need to check for duplicates when adding to the array - so that the ID in the array isn't duplicated, further preventing restart. 

A solution to the root problem will come in time, but this is the first step.

TLDR this is a ramble I've written to myself to remember to check for duplicates when adding to the playing servers array.

**Comment by R2D2VaderBeef** [comment]
I've pushed the check for duplicates, and a possible fix to remove any not-playing servers from the array when a restart is requested. Testing required, I'm not sure of the repercussions of this.

**Comment by R2D2VaderBeef** [comment]
I think this is probably fixed.
